lib/ci/build-types/citgm_comparison_build.js

```diff
@@ -33,6 +34,14 @@ class CITGMComparisonBuild {
     await baseBuild.getResults();
     await comparisonBuild.getResults();
 
+    if (baseBuild.failures.length || comparisonBuild.failures.length) {
+      this.failures.baseBuild =
+        this.failures.baseBuild.concat(baseBuild.failures);
+      this.failures.comparisonBuild =
+        this.failures.comparisonBuild.concat(comparisonBuild.failures);
```

Comment:
Nit: not sure which one is better, but this one is shorter :smile: 
```suggestion
      this.failures.baseBuild.push(...baseBuild.failures);
      this.failures.comparisonBuild.push(...comparisonBuild.failures);
```